jordan_bundy/lib/greet.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var exports = module.exports = {};
+
+var greet = function greet(name) {
+  
+  // Not pretty, but a quick and dirty way to use process.argv
+  var name = name || process.argv;
```

Comment:
This looks like you are overthinking things - what if your function is just 
`var greet = function(name) {
  return 'hello ' + name;
};`

This will give you the same functionality in fewer lines of code.